be/src/storage/lake/compaction_policy.cpp

```diff
@@ -39,6 +39,40 @@ class BaseAndCumulativeCompactionPolicy : public CompactionPolicy {
     TabletMetadataPtr _tablet_metadata;
 };
 
+class SizeTieredCompactionPolicy : public CompactionPolicy {
+public:
+    explicit SizeTieredCompactionPolicy(TabletPtr tablet)
+            : CompactionPolicy(tablet),
+              _max_level_size(config::size_tiered_min_level_size *
+                              pow(config::size_tiered_level_multiple, config::size_tiered_level_num)) {}
+    ~SizeTieredCompactionPolicy() override = default;
+
+    StatusOr<std::vector<RowsetPtr>> pick_rowsets(int64_t version) override;
+
+private:
+    double cal_compaction_score(int64_t segment_num, int64_t level_size, int64_t total_size, KeysType keys_type,
```

Comment:
turn to a static function?

Reply:
i see `_max_level_size` is a read only var in the function body. how about move it to function paramter so the function itself can be completely static.

a static function is preferred because a) it is a procedure oriented, no need to understand the related class internal logic, b) it has deterministic input and output, easy to cover by unit test.

Reply:
ok